database/redis/notification.go

```diff
@@ -133,6 +133,115 @@ func (connector *DbConnector) removeNotifications(notifications []*moira.Schedul
 	return total, nil
 }
 
+// GetDelayedTimeInSeconds returns the time, if the difference between notification
+// creation and sending time is greater than this time, the notification will be considered delayed
+func (connector *DbConnector) GetDelayedTimeInSeconds() int64 {
+	return int64(connector.notification.DelayedTime.Seconds())
+}
+
+// filterNotificationsByDelay filters notifications into delayed and not delayed notifications
+func filterNotificationsByDelay(notifications []*moira.ScheduledNotification, delayedTime int64) ([]*moira.ScheduledNotification, []*moira.ScheduledNotification) {
```

Comment:
Maybe we shouyld use named returns here? It would be painfullty non obvious which returned walue is delayed and which is is not delayed just by signature
```suggestion
func filterNotificationsByDelay(notifications []*moira.ScheduledNotification, delayedTime int64) (delayedNotifications []*moira.ScheduledNotification, notDelayedNotifications []*moira.ScheduledNotification) {
```

Reply:
Added, it came out a bit verbose, but in this PR #952 I'll fix that problem